tests/wicked/barrier_init.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Standalone card - ifdown, ifreload
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use lockapi;
+use mmapi;
+
+sub run {
+    my ($self, $args) = @_;
+
+    my $children     = 2;
```

Comment:
Why this hardcoding? Doesn't it brake the `keys` call?

Reply:
indeed, fixing it. What do you prefer, guys? 

    my $children     = get_children();
    my $num_children = scalar(keys %$children) + 1;    # +1 for it self

or

    my $num_children = 2;

Reply:
I prefer the dynamic way

Reply:
**hpc** has a vairable for it `CLUSTER_NODES` and **ha** a more complex one `CLUSTER_INFOS`.
I don't have hard feelings regarding this. If you think we should go with static  lets do it. And we can make it dynamic or configurable when needed.

Reply:
we leave it as it is, and let's see if it makes sense to change it later on to a given variable.